models/src_apple_store.yml

```diff
@@ -7,7 +7,7 @@ sources:
     loader: Fivetran
     loaded_at_field: _fivetran_synced
 
-    freshness:
+    freshness: 
```

Comment:
Just wanted to check that we do in fact want this freshness range to be applied to all of the source tables.

Reply:
Thanks for the check in Joe! This actually brings me to a follow up question - does freshness apply to the refresh of the actual source data or the refresh of the tables after Fivetran has synced them? I know that the Apple Store (itunes) connector can be reaaaaaally slow, so I'm wondering if this affects the freshness. If it's the latter, I think this is a good starting point for freshness.

Reply:
Great question! Freshness applies to the source data itself and how fresh the data is (ie. how recently the last sync of data occurred). If we apply a freshness test to error after 72 hours, we are asking the package to throw an error on `dbt source freshness` if the package doesn't see any new data in the source table after a period of 72 hours. Based on this config we are checking the `_fivetran_synced` field to do this check.

Based on our experience with the connector I would suggest a freshness of the below config:
```yml
      warn_after: {count: 48, period: hour}
      error_after: {count: 168, period: hour}
```

I suggest this because the connector should be syncing quite frequently and I would like to warn users if the connector syncs are longer than two days. Additionally, I have seen a few syncs take almost a week. I would prefer we warn users at that point as the connector really needs to be looked at to investigate further at that point.

Let me know if this helps or if you have any more questions 😄